src/styles/main.css

```diff
@@ -1,93 +1,228 @@
     cursor: pointer;
+    user-select: none;
+}
+
+.columnBody {
+    padding: 0px 3px;
+    position: relative;
+    z-index: 2;
+}
+
+/* End column component */
+
+/* Card component */
+
+.cardContainer {
+    border-radius: 2px;
+    border-left: 3px solid #cc3366;
+    box-shadow: 0 1px 2px 0 rgb(9 30 66 / 25%);
+    height: 110px;
+    margin: 7px 2px;
+    background-color: #fff;
+    cursor: move;
+    transition: 0.3s all;
+    padding: 10px 15px;
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+}
 
-    transition: background-color 150ms ease;
+.cardContainer:hover {
+    background: #f5f5f5;
+    border-color: #999;
+    border-left: 3px solid #cc3366;
 }
 
-button:hover {
-    background-color: var(--button-hover-color)!important;
+.addColumnBtn {
+    display: none;
+    z-index: 10;
 }
 
-.logo {
-    margin: 20px;
+/* Ticket info */
+#ticketWrapper {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0px;
+    background-color: #172b4d87;
+    display: none;
+    z-index: 20;
+}
+#ticketInfoContainer {
+    position: relative;
+    width: 40%;
+    height: 600px;
+    top: 30px;
+    left: 50%;
+    transform: translate(-50%, 0px);
+    background-color: #fff;
+    border-radius: 3px;
+    z-index: 30;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    padding: 20px 35px;
+}
+
+#ticketInfo {
+    margin-top: 40px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    height: 70%;
 }
 
-.secondary-color {
-    color: var(--secondary-color);
+#ticketInfoButtons {
+    display: flex;
+    width: 100%;
+    justify-content: flex-end;
+    gap: 10px;
 }
 
-.decoration {
-    animation: rotate 2000ms linear infinite;
+.createTicket {
+    border: none;
+    margin-right: 4px;
+    height: 32px;
+    min-width: 24px;
+    border-radius: 3px;
+    text-shadow: none;
+    line-height: 32px;
+    padding: 0 10px;
+    color: #fff;
+    background: #0052cc;
+    cursor: pointer;
+    transition: 0.3s all;
 }
 
-.button, .button-cta {
-    transition: transform 500ms cubic-bezier(1, 0.01, 0.57, 1.35);
+.cancelTicket {
+    border: none;
+    margin-right: 4px;
+    height: 32px;
+    min-width: 24px;
+    border-radius: 3px;
+    text-shadow: none;
+    line-height: 32px;
+    padding: 0 10px;
+    color: #0052cc;
+    background-color: #fff;
+    cursor: pointer;
+    transition: 0.3s all;
+}
+
+.createTicket:hover {
+    background-color: #0065ff;
+}
+
+.cancelTicket:hover {
+    color: #0065ff;
+    text-decoration: underline;
+}
+
+.add {
+    cursor: pointer;
+}
+
+h2 {
+    width: 100%;
+    font-weight: normal;
+    font-size: 20px;
+    margin: 0;
+    color: #172b4d;
 }
 
-.active {
-    transform: translateY(50%);
+.issueType {
+    height: 20px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, "Fira Sans", "Droid Sans",
```

Comment:
font-xxx should be defined on the body as a constant for all the page. From there with regards to inheritance we can overwrite them where it is needed.